### PR TITLE
Update create-and-manage-dial-plans.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-and-manage-dial-plans.md [hunk 1]
```diff
@@ -81,12 +81,13 @@ To learn more, see [Connect to all Microsoft 365 or Office 365 services in a sin
     
 2. In the **Windows PowerShell** window, connect to Microsoft 365 or Office 365 by running:
     
+ 
     > [!NOTE]
```
Comment (JohanFreelancer9) [inline Teams/create-and-manage-dial-plans.md:85]:
```suggestion
   > [!NOTE]
```

File: Teams/create-and-manage-dial-plans.md [hunk 2]
```diff
@@ -81,12 +81,13 @@ To learn more, see [Connect to all Microsoft 365 or Office 365 services in a sin
...
-    > You only have to run the **Import-Module** command the first time you use the Skype for Business Online Windows PowerShell module.
-  
+    > Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/create-and-manage-dial-plans.md:86]:
```suggestion
   > The Skype for Business Online Connector is currently part of the latest Teams PowerShell Module.
```

File: Teams/create-and-manage-dial-plans.md [hunk 3]
```diff
@@ -81,12 +81,13 @@ To learn more, see [Connect to all Microsoft 365 or Office 365 services in a sin
...
+    > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Teams/create-and-manage-dial-plans.md:87]:
```suggestion
  > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**